test/ToolsTestFileIO.cpp

```diff
@@ -642,15 +643,48 @@ TEST_F(TestFileIO, TestReadAsciiFileContentAsRoot) {
    ASSERT_TRUE(badResult.HasFailed());
 
    //Open a common root permissioned file.
-   auto goodResult = FileIO::ReadAsciiFileContentAsRoot("/etc/sysconfig/iptables-config");
+   std::string filePath("/etc/sysconfig/iptables-config");
+   auto goodResult = FileIO::SudoFile(FileIO::ReadAsciiFileContent, filePath);
    EXPECT_FALSE(goodResult.HasFailed());
    EXPECT_TRUE(goodResult.result.size() > 0);
+
+   FileIO::SetUserFileSystemAccess("root");
 }
 
+TEST_F(TestFileIO, TestSudoFileRemoveFile) {
+
+   // Start as root user
+   int previousUID = setfsuid(-1);
+   int previousGID = setfsgid(-1);
+   ASSERT_EQ(previousUID, 0);
+   ASSERT_EQ(previousGID, 0);
+
+   std::string filename{"/tmp/TestFileIO_"};
```

Comment:
I agree. I changed this to use the mTestDirectory which is already setup and cleaned up.